pages/learn/deploy/release-strategy/release-policy.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Release policy
+excerpt: Manage the application and device release policy
+---
+
+# Release policy
+
+When managing an application fleet, you may require devices to be running different releases. By default, new releases are deployed to all devices in the fleet once successfully built. However, you can customize this behavior, so the application fleet or individual devices remain on a fixed release by pinning them.
+
+You may define the application and device release policy via the [{{ $names.cloud.lower }} dashboard]({{ $links.dashboardUrl }}) or programmatically through the [API][api] or [SDKs][sdk].
+
+__Note__: It is currently not possible to specify the application or device release policy via the {{ $names.cli }}.
```

Comment:
is there an issue tracking this? might be worthwhile to link to here.

Reply:
@xginn8 i don't think there is a specific one for this one, but there are couple like https://github.com/balena-io/balena-cli/issues/1499 which talk about adding the notion of releases to the cli and perhaps we should do this with all that work?

Reply:
that sgtm, i just wanted an easy place to aggregate the :+1: 's

Reply:
@xginn8 although thinking about it a bit, probably creating a specific issue for this would be good. Im not sure how the UX would work because it would be something like `balena device pin <uuid> <release>` and `balena app pin <appID>`... but we could just as easily have a `balena pin <app_or_device> <release>` command, worth some discussion.